src/pages/activity/activity-or-route-planner/activity-duration/ActivityDuration.tsx

```diff
@@ -148,7 +147,20 @@ const ActivityDurationPage = () => {
         return skip;
     };
 
+    const isCompleteTimeframe = () => {
```

Comment:
Cette fonction s'appelant "is...." on s'attendrais à ce qu'elle renvoi un booléen. Son nom ne me semble pas adapté.